### PR TITLE
update algolia guide

### LINKED ISSUE #1780: Update Migrating from Algolia to Meilisearch guide to Meilisearch v0.28
We should update the migration guide to have latest version of Meilisearch (v0.28)

### PR DESCRIPTION
**Pull request by maryamsulemani97**
closes #1780

### CODE REVIEW

**Review by guimachiavelli — APPROVED** [review]
I can't make a suggestion, but on L#59 we're using meilisearch-js v0.25.1, but the most recent seems to be v0.27.0. Not a huge deal, but might as well update that too?

File: learn/getting_started/algolia_migration.md [hunk 1]
```diff
@@ -22,10 +22,10 @@ Before continuing, make sure you have both Meilisearch and Node.js installed and
 
 This guide was tested with the following package versions:
 
-[`node.js`](https://nodejs.org/en/): `16.14`
+[`node.js`](https://nodejs.org/en/): `16.16`
 [`algoliasearch`](https://www.npmjs.com/package/algoliasearch): `4.13`
 [`meilisearch-js`](https://www.npmjs.com/package/meilisearch): `0.25.1`
```
Comment (maryamsulemani97) [inline learn/getting_started/algolia_migration.md:27]:
```suggestion
[`meilisearch-js`](https://www.npmjs.com/package/meilisearch): `0.27.0`
```

**Review by guimachiavelli — APPROVED** [review]
🌈

**Comment by maryamsulemani97** [comment]
bors merge

**Pull request merged**